test/a11y/AccessibilityProblemPageSpec.scala

```diff
@@ -37,11 +37,9 @@ class AccessibilityProblemPageSpec
     with MessagesSupport
     with AccessibilityMatchers {
 
-  implicit lazy val fakeRequest: RequestHeader = FakeRequest("GET", "/foo").withCSRFToken
-
-  implicit lazy val messages: Messages = getMessages(app, fakeRequest)
-
-  implicit lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
+  given fakeRequest: RequestHeader = FakeRequest("GET", "/foo").withCSRFToken
+  given Messages                   = getMessages()
+  given AppConfig                  = app.injector.instanceOf[AppConfig]
```

Comment:
this boilerplate is repeated across all the a11y and view specs - could maybe extract a base spec for these, but didn't want to conflate with this PR

Reply:
One for tech debt?